src/Networks/Blockcore.Networks.XRC/Consensus/XRCConsensusFactory.cs

```diff
@@ -10,15 +10,11 @@ public class XRCConsensusFactory : ConsensusFactory
     {
         public XRCConsensusFactory() : base()
         {
-            this.Protocol = new ConsensusProtocol();
```

Comment:
It is correct because Protocol is set later in network configuration.
https://github.com/block-core/blockcore/blob/master/src/Networks/Blockcore.Networks.XRC/XRCMain.cs#L100

```
            consensusFactory.Protocol = new ConsensusProtocol()
            {
                ProtocolVersion = ProtocolVersion.FEEFILTER_VERSION,
                MinProtocolVersion = ProtocolVersion.POS_PROTOCOL_VERSION,
            };
```